pixeldata/src/encapsulation.rs

```diff
@@ -0,0 +1,207 @@
+use dicom_core::value::{Value, C};
+use dicom_core::DataDictionary;
+use dicom_object::mem::InMemFragment;
+use dicom_object::InMemDicomObject;
+
+#[derive(Debug)]
+pub struct EncapsulatedPixels {
+    bot: C<u32>,
+    current_offset: u32,
+    fragments: C<Vec<u8>>,
+    number_of_fragments: u32,

```

Comment:
It is not clear what this field is for. Is it the number of fragments per frame? I feel that the frame splitting criterion should rather be based on a maximum fragment size in bytes.